opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -37,50 +37,46 @@
 ValueT = TypeVar("ValueT", int, float)
 
 
-class DefaultBoundInstrument:
-    """The default bound metric instrument.
-
-    Used when no bound instrument implementation is available.
-    """
-
+class BoundCounter(abc.ABC):
     def add(self, value: ValueT) -> None:
-        """No-op implementation of `BoundCounter` add.
+        """Increases the value of the bound counter by ``value``.
 
         Args:
-            value: The value to add to the bound metric instrument.
+            value: The value to add to the bound counter. Must be positive.
         """
 
-    def record(self, value: ValueT) -> None:
-        """No-op implementation of `BoundValueRecorder` record.
-
-        Args:
-            value: The value to record to the bound metric instrument.
-        """
 
-    def release(self) -> None:
-        """No-op implementation of release."""
+class DefaultBoundCounter:
```

Comment:
Shouldn't this class inherit from `BoundCounter`?

Reply:
Good catch.